brave-resteasy3-spring/pom.xml

```diff
@@ -24,8 +24,10 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <resteasy.version>3.0.8.Final</resteasy.version>
-    <jetty.version>8.1.9.v20130131</jetty.version>
-     <spring.version>3.2.2.RELEASE</spring.version>
+      <jetty.version>8.1.9.v20130131</jetty.version>
+      <!--<jetty.version>9.3.0.M1</jetty.version>-->
+     <spring.version>4.0.5.RELEASE</spring.version>
+          <maven.test.skip>true</maven.test.skip>
```

Comment:
Sorry for that, that change has sneaked into my commits. I had a test collision with our production tests possibly because of usage of the same ports or something like this if I remember right. This file changes are not to be included.